src/components/CreateEventAttestationButton.tsx

```diff
@@ -0,0 +1,42 @@
+import { useCreateEventAttestation } from "../hooks/useCreateEventAttestation";
+import type {Event} from "../hooks/useCreateEventAttestation"
+
+export function CreateEventAttestationButton() {
+    const createEventAttestation = useCreateEventAttestation();
+
+    async function handleCreateEventAttestation() {
+
+        const today = new Date();
+        const tomorrow = new Date();
+        tomorrow.setDate(today.getDate()+1);
```

Comment:
please lint/format your code when submitting a PR -- not relevant here but we don't want to review formatting changes down the line